data/remnant/remnant jobs.txt

```diff
@@ -659,3 +659,44 @@ mission "Remnant: Rescue 5"
 	on visit
 		dialog
 			`You have returned, but there is still no word from the missing ship.`
+
+
+mission "Remnant: Disable 1"
+	job
+	repeat
+	name "Disable Damaged Kas'lor Ik 582"
+	description "A damaged Kas'lor Ik 582 is stuck in the Ember Waste sectors bordering Korath space. It has a large civilian population so we ask you to only disable it to avoid an atrocity. Once you return to <planet> you will receive a logistics adjustment of <payment>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Will Not Someone Please Think Of The Children: done"
+		random < 5
+	npc disable save
+		government "Korath"
+		personality heroic vindictive target marked staying
+		system 
+			near "Vaticanus" 1
+		fleet
+			names "korath"
+			variant
+				"Kas'lor Ik 582 (Stranded)"
+		dialog `You have disabled the Kas'lor Ik 582, time to report back to <planet>.`
+	on visit
+		dialog "You've landed on <planet>, but the stuck Kas'lor Ik 582 has not been disabled yet. Hunt it down and disable it before returning."
+	on abort
+		"reputation: Remnant" += 250
+	on fail
+		"reputation: Remnant" -= 250
+		conversation
+			branch failed
+				not "Remnant: Disable 1: aborted"
+			action
+				clear "Remnant: Disable 1: aborted"
+			`The remnants are puzzled at your aborting of this mission, but they tell you they will send someone else to do this work.`
```

Comment:
`remnants`?
Should probably start with a capital and perhaps not have an `s` at the end.